docs/src/dev-manual/devel.md

```diff
@@ -208,13 +212,11 @@ sure you update `/path/to/enduro` to the proper project folder in the host):
 - Host path: `/path/to/enduro/hack/sampledata/StructB-AM.zip`
 - Object name: `StructB-AM.zip`
 
-Alternatively, you can use the Enduro API to upload the file like in the
-following example:
+Alternatively, run the following make thing to submit a transfer via the
+`/package/upload` API via cURL:
```

Comment:
`make thing`?  :smile:

Reply:
I couldn't remember what was the actual name... goals, recipes, targets? And then I forgot to look it up :)